src/hidpp20.c

```diff
@@ -1012,6 +1014,76 @@ hidpp20_color_led_effect_get_zone_effect_info(struct hidpp20_device *device,
 	return 0;
 }
 
+int
+hidpp20_color_led_effects_set_zone_effect(struct hidpp20_device *device,
+					  uint8_t zone_index,
+					  struct hidpp20_led led)
```

Comment:
I feel like I'm missing something here - there is no argument for the zone effect, what are we actually setting here?

Reply:
Yes there is, `struct hidpp20_led` stores the led settings, including the effect.